test/helpers/kubectl.go

```diff
@@ -1842,6 +1879,56 @@ func (kub *Kubectl) CiliumServicePreFlightCheck() error {
 	return nil
 }
 
+// DeployETCDOperator deploys the etcd-operator k8s descriptors into the cluster
+// pointer by kub.
+func (kub *Kubectl) DeployETCDOperator() error {
+	const etcdOperatorPath = "../examples/kubernetes/addons/etcd-operator/"
+	deployFile := func(filename string) error {
+		cmdRes := kub.Apply(GetFilePath(etcdOperatorPath + filename))
+		if !cmdRes.WasSuccessful() {
+			return fmt.Errorf("Unable to deploy descriptor of etcd-operator %s: %s", filename, cmdRes.OutputPrettyPrint())
+		}
+		return nil
+	}
+
+	kub.Exec(fmt.Sprintf("%s '%s'", GetFilePath(etcdOperatorPath+"tls/certs/gen-cert.sh"), "cluster.local"))
```

Comment:
Should not these two commands return error if fails?

Reply:
Fixed